src/Form/AdminSettingsForm.php

```diff
@@ -68,7 +61,7 @@ public static function create(ContainerInterface $container) {
   /**
    * {@inheritdoc}
    */
-  public function buildForm(array $form, FormStateInterface $form_state, Request $request = NULL) {
+  public function buildForm(array $form, FormStateInterface $form_state) {
```

Comment:
I did not include descriptions in this story, because I wanted to focus the scope on the essentials and move on quickly. I agree description texts are helpful and easy, so I will add them as well.

As a customer, I do not feel that I need to know the exact default value, as long as I know I can safely leave some form fields empty. What I need to know is that I was told a list of items to be filling into the form, and I can do so without confusion or hassle. I would also avoid tech details so customers are required to learn less.

Pre-filled forms sounds great. It feels like a nice-to-have instead of an essential, so I would recommend that we do this in a separate story. I can certainly create a story to track this. Wanna do that?